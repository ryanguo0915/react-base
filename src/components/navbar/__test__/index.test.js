import React from 'react';
import { shallow } from 'enzyme';
import NarBar from '../index';
describe("NarBar", () => {
  it("should render my component", () => {
    const props = { 
      site: {
        title: 'Title',
        imageLogo: ''
      },
      firstName: 'John'
    };
    const wrapper = shallow(<NarBar {...props}/>);
    expect(wrapper.getElements()).toMatchSnapshot();
  });
});