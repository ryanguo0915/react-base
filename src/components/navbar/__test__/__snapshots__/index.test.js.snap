// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NarBar should render my component 1`] = `
Array [
  <div
    className="navbar"
  >
    <Link
      className="navbar__avatar"
      to="/"
    >
      <img
        alt="fireSpot"
        height={30}
        width={30}
      />
    </Link>
    <h2
      className="navbar__title"
    >
      Title
    </h2>
    <span
      className="navbar__profile"
    >
      <Link
        to="/profile"
      >
        Welcome 
        John
      </Link>
    </span>
  </div>,
]
`;
